frontend/src/metabase/entities/schemas.js

```diff
@@ -180,3 +193,42 @@ function addTableAvoidingDuplicates(tables, tableId) {
   }
   return tables.includes(tableId) ? tables : [...tables, tableId];
 }
+
+const useGetQuery = ({ id, ...args }) => {
```

Comment:
This hook mimics this: https://github.com/metabase/metabase/blob/ba3b7818e6cdf155a74bf00f2b3310c9eec616fc/frontend/src/metabase/entities/schemas.js#L64-L90